### ISSUE TITLE
Mobile VO, magnet is "released" as soon as it is picked up

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/faradays-law#206]
**Device**
iPad 6th Gen
**OS**
iPadOS 14.2
**Browser**
Safari
**Problem Description**
For https://github.com/phetsims/QA/issues/581
When using mobile VO and focusing on the magnet, it will tell you to double tap and drag, then let go to stop. When you do this, it will immediately say "Release" and play the release sound effect. You can still drag the magnet around however. I also noticed that the instructions will include things like using the wasd keys and the 123 autoslide options, despite this being the iPad version of the sim with no keyboard nav dialog.
**Visuals**
https://drive.google.com/file/d/1YwF5N1ylZp44sAvyGxTL8xZIzKY9epXH/view?usp=sharing

**Comment by KatieWoe** [comment]
Further note. When you let go of the magnet it is difficult to pick up again. If you try to do the double tap the magnet to pick it up you get an error "donk" sound. To pick it up you need to move to another bit of description by swiping then coming back to the magnet. I also noticed that the bit where it says "Double tap to pick up" is not the focus where you need to do that.

**Comment by jessegreenberg** [comment]
@jbphet and I reviewed this. He is able to produce this on iOS 14. I am not able to produce this on iOS 13.6. So I think this is related to https://github.com/phetsims/scenery/issues/1094, and I hope that a solution to that issue will resolve this as well.

**Comment by jbphet** [comment]
I've decided that this bug should not block publication of the 1.4 release.  It only occurs on iOS devices when using VoiceOver, and doesn't make the sim unusable, just a bit awkward.  Once a fix is available for this, please assign the issue back to me and I'll do a maintenance release from the 1.4 branch with the changes.

**Comment by jessegreenberg** [comment]
This should be fixed by https://github.com/phetsims/scenery/issues/1137, that issue is currently ready for review.

**Comment by jessegreenberg** [comment]
This was fixed in github.com/phetsims/scenery/issues/1137. I don't think a maintenance release is necessary for this sim though @jbphet since description was not part of the recent release. Specifically, there are no alerts that describe position of the magnet so this wouldn't be very useful for someone actually using VoiceOver yet. I think it is OK to close, but feel free to double check with @terracoda or @emily-phet first if you wish.

**Comment by terracoda** [comment]
I double checked. The magnet is indeed grabbable and draggable with touch and mobile VO.

**Comment by terracoda** [comment]
Though the state descriptions are not perfect, and there are no responses, this sim works provides a relatively rich non-visual experience when the sound and sonification is combined with accessible interaction names provided via iOS VoiceOver.

I don't know what it takes to do a maintenance release, but if it isn't too much trouble, I would encourage it. 

Otherwise, we can do a dev version, and I can update the prototype version on the Accessible Sims page.

We could also do both, but I am perfectly happy with a dev version that I can provide on the A11y Sims page.

**Comment by terracoda** [comment]
I just used the published version of the sim, and the though the behaviour is not perfect, it is possible to grab the magnet with VO enabled, both on iOS and on MacOs.

Let's leave it for now and make sure our best version with interactive description (and with the interactive description icon) is available via a dev version in the list of prototypes on the A11y Sims page.

The version with interactive description is still a prototype. I think that path makes sense.

**Comment by terracoda** [comment]
@jessegreenberg, can you give me a dev version here, and I will create a prototype entry on the web page.

**Comment by terracoda** [comment]
Oh maybe, it is better if @jbphet gives me the latest dev version. The issue is assigned to him.
@jbphet, please give me the latest dev version and I will close this issue :-)

**Comment by jbphet** [comment]
@terracoda - The latest version is https://phet-dev.colorado.edu/html/faradays-law/1.4.0/phet/faradays-law_en_phet.html.

**Comment by terracoda** [comment]
@jbphet, sorry I meant a "new dev version" which includes @jessegreenberg Jan 13th commit that fixes the magnet.
That dev version 1.4.0 is from December 23rd.

Please generate a new dev version. Apologies for not being clear.

**Comment by jbphet** [comment]
@terracoda - Thanks for the clarification.  I've published https://phet-dev.colorado.edu/html/faradays-law/1.5.0-dev.1/phet/faradays-law_en_phet.html for you to check out.  That should be the "latest and greatest" as of this moment.

**Comment by terracoda** [comment]
Thanks @jbphet I am able to grab and drag the magnet with mobile VO on iOS 14.3. 

However, I am sorry to report that with that version I can no longer move the grabbed magnet with the keyboard. On MacOS 10.13.6 with or without VoiceOver enabled and browsing with Safari the grabbed magnet is not at all operational.

The visual grab cue is also askew.
Maybe something happened in common code? There has been some recent work to fix iOS 14 issues.

The published version works fine with the keyboard.

I'll open a new issue for the keyboard problem, and wait to update the link on eh A11y Sims page.

**Comment by terracoda** [comment]
I think it is safe to close this issue as the magnet is now operational on Mobile VO.
